toolkit/templates/contact-details.html

```diff
@@ -28,28 +28,38 @@
   {% endif %}
 {% endif %}
 {% if
-  street_address is defined or
-  locality is defined or
-  postcode is defined
+  street_address or
+  locality or
+  country or
+  postcode
 %}
   <p class="contact-details-block{% if without_spacing %}-without-spacing{% endif %}" itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
-    {% if street_address is defined %}
+    {% if street_address %}
     <span itemprop="streetAddress">
-      {{ street_address }}
+      {{ street_address.line_1 }}
```

Comment:
Can we split `street_adress` into two variables instead? That would match two separate fields in our supplier data: `.address1` and `.address2`. I'm not sure if we can assign dictionaries in with statements to get around this, but I don't think we have an object with `.line_1` and `.line_2` anywhere so it might be more work anyway.